docs/bigcommerce-for-wordpress/extending-the-plugin/bigcommerce-for-wordpress-cron.md

```diff
@@ -0,0 +1,60 @@
+<div><h3 class="sub-docs-type" id="bigcommerce-for-wordpress">BigCommerce for WordPress</h3></div>
+
+# BigCommerce for WordPress Cron
+
+This article provides a general description for WP-Cron and how to change its behavior in order to make it more reliable.
+
+BigCommerce for WordPress plugin has update/sync logic which is based on cron jobs (WP-Cron). WP-Cron is built-in logic for processing postponed, background tasks. While that logic is great and flexible it also has some problems which may affect import product logic.
+
+## Wp-Cron vs Server Cron
+WordPress has its own cron system for scheduling tasks such as checking for updates, scheduling a post for publication, and deleting comments from the trash. All cron jobs are handled by WP-Cron.
+
+Meanwhile the logic has the name WP-Cron, it is not a real server cron job. The difference is:
+- Server cron runs automatically and is fired by server processes. There is no need for an external source for events firing
+- WP-Cron runs only when someone is on the site
+
+<p>Here's an example:</p>
+
+You schedule a job to run once an hour with WP-cron and WordPress will set an execution time for that job. However, the job won't be performed until there's site-traffic or there is an external request to WordPress. 
+
+Because of this, WP-Cron jobs may be postponed or not performed at all. On the other hand, if you've set a job via the usual server cron it will be performed even if there isn't site traffic or external requests.
+
+## WP-Cron enhance with Server Cron
+In order to make WP-Cron more reliable and not dependent on site-traffic, the following steps should be taken:
+
+### Open and edit wp-config.php
+1. Open the wp-config file using an FTP client like [FileZilla](https://filezilla-project.org/) or SSH client.
+2. Add the following string to the file:`define('DISABLE_WP_CRON', true);`
+3. This code disables the running cron events on your WordPress site and allows you to create a real cron job.
+
+### Add new cron job to the server
+The following are different methods that can be used for adding a new cron job to the server.
+
+**Cron job manager**
+
+There are plenty of job managers available for hosting providers like [HostGator](https://www.hostgator.com/) or [Cloudways](https://www.cloudways.com/en/).
+
+>**Note:** We recommend that you contact the hosting’s tech support for proper setup. Each hosting provider has specific detailed knowledge bases that cover how to set up cron jobs.
+
+The following example is for HostGator but it also can be used for any other hosting with CPanel: 
+1. Login to CPanel.
```

Comment:
```suggestion
1. Log in to cPanel.
```
- cPanel is the official spelling of the product: https://cpanel.net/
- "Log in" is the verb form. Google style guide prefers "sign in" over "log in," but that is totally optional. "login (noun or adjective), log in (verb). For the verb form, _sign in_ is generally better, but if you're documenting a tool that uses the term _log in_, then use the term that the tool uses."